usb/audio_hal.c

```diff
@@ -1368,12 +1368,12 @@ void stop_existing_output_input(struct audio_device *adev){
 
     list_for_each(node, &adev->output_stream_list) {
         struct audio_stream* stream = (struct audio_stream *)node_to_item(node, struct stream_out, list_node);
-        out_standby((struct audio_stream_out *)stream);
+        out_standby((struct audio_stream *)stream);
```

Comment:
I think it's better we use the struct member, instead of type casting ? i.e.
```suggestion
        out_standby(&stream->common);
```

Reply:
@akodanka ,The stream is already of type "struct audio_stream" and out_standby expects it to be struct audio_stream", actually typecast itself is not required in this case. Since it was wrongly typecast to "audio_stream_out", i changed it to "struct audio_stream".